### PR TITLE
KEYCLOAK-14495 Introduce support for JAVA_OPTS_APPEND

### PR DESCRIPTION
**Pull request by radudd**
This is a proposal to introduce support for JAVA_OPTS_APPEND environment variable. The corresponding JIRA ticket: https://issues.redhat.com/browse/KEYCLOAK-14495

### CODE REVIEW

File: server/tools/docker-entrypoint.sh [hunk 1]
```diff
@@ -104,6 +104,9 @@ if echo "$@" | grep -E -v -- '-c |-c=|--server-config |--server-config='; then
     SYS_PROPS+=" -c=standalone-ha.xml"
 fi
 
+# Adding support for JAVA_OPTS_APPEND
+sed -i '$a\\n# Append to JAVA_OPTS. Necessary to prevent some values being omitted if JAVA_OPTS is defined directly\nJAVA_OPTS=\"\$JAVA_OPTS \$JAVA_OPTS_APPEND\"' /opt/jboss/keycloak/bin/standalone.conf
```
Comment (slaskawi) [inline server/tools/docker-entrypoint.sh:108]: This seems to be very fragile. Perhaps you could add this at the end of the file?

Comment (radudd) [reply server/tools/docker-entrypoint.sh:108]: Thanks @slaskaw for reviewing. Adding support for this option will allow to add profile features to Keycloak https://www.keycloak.org/docs/latest/server_installation/#profiles. For my particular scenario, I would need to enable upload_script profile.

Comment (radudd) [reply server/tools/docker-entrypoint.sh:108]: > This seems to be very fragile. Perhaps you could add this at the end of the file?
Yes, I agree. But the standalone.conf file is not part of the repo and is extracted from the Keycloak binary which is downloaded and installed during the build.

Comment (slaskawi) [reply server/tools/docker-entrypoint.sh:108]: Ok, got it.

**Comment by radudd** [comment]
Thanks @iankko and @slaskawi for the review and feedback. I just added an example on how to use JAVA_OPTS_APPEND

**Review by slaskawi — APPROVED** [review]
LGTM, let's give @iankko a chance to look into it.

**Review by iankko — APPROVED** [review]
/lgtm Thanks! @radudd 

(having the container image with the change built as with `0552f7ce2dc0`, and tried to enable
 e.g. the new account console via:

 ```
     $ docker run -it \
     -e KEYCLOAK_USER=admin \
     -e KEYCLOAK_PASSWORD=admin \
     -e JAVA_OPTS_APPEND="-Dkeycloak.profile=preview" \
     0552f7ce2dc0
````
the change works fine / as expected (`keycloak-preview` appears as configurable account theme &
once used / saved, its displayed when the user goes to account management).

**Comment by slaskawi** [comment]
Integrated, thanks @radudd

**Pull request merged**